CONTRIBUTING.md

```diff
@@ -60,9 +60,22 @@ This command generates static content in the `build/` directory and can be serve
 ### Deployment
 
 To build and deploy to GitHub Pages, use:
+1. Follow step 1 - 5 in [Docusaurus's Triggering deployment with GitHub Actions](https://docusaurus.io/docs/deployment)
 
+2. Modify `docusaurus.config.js` and change `organizationName: 'Seneca-ICTOER'` to `  organizationName: 'GITHUB_USERNAME'`
```

Comment:
> i.e., do we need to do this in our case, or are you documenting this for people setting it up on their forks?

yes, I'm documenting so that people can deploy locally on their forks

Reply:
OK.  Because that's not our primary use case here, I'd suggest you add a note that indicates that it's only necessary if you're testing on a fork.  Otherwise, focus on what this site/repo needs to do.